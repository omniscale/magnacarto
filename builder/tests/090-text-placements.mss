@foo: 2;

#labels {
    text-name: "[name]";
    text-fill: black;
    text-size: 12;

    text-placement-list: {
        text-dx: 6+3;
    },{
        text-size: @foo * 3;
        text-name: "[nm]"
    };
}
