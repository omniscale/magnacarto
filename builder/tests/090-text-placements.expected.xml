<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="labels" filter-mode="first">
    <Rule>
      <TextSymbolizer fill="#000000" placement-type="list" size="12">[name]
        <Placement dx="9"></Placement>
        <Placement size="6">[nm]</Placement>
      </TextSymbolizer>
    </Rule>
  </Style>
  <Layer name="labels" srs="" status="off">
    <StyleName>labels</StyleName>
  </Layer>
</Map>